util/containerutil/alias_test.go

```diff
@@ -0,0 +1,19 @@
+package containerutil_test
+
+import (
+	"github.com/stretchr/testify/require"
```

Comment:
I kind of prefer `require` over `assert` as those panics can be a hassle.

Reply:
me too -- that's why this PR exists :)